src/Input.jsx

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+
+//Use 
+//<Input label="name:" type="text" etc /> 
```

Comment:
And here

Reply:
This is just a comment to remind us how to use the input component :) we will delete it once we have properly implemented it!